Packs/DuoAdminApi/ReleaseNotes/3_0_2.md

```diff
@@ -0,0 +1,10 @@
+
+#### Integrations
+##### DUO Admin
+ added 4 commands and modified one command
```

Comment:
I dont understand the comment here.  I submitted the pack update request through XSOAR

Reply:
Because this is an XSOAR-supported pack, we must follow the conventions when writing the release note.
I sent you a [link](https://xsoar.pan.dev/docs/documentation/release-notes) to our doc, the prefix for each row must start with the ones presented in the doc, such as:
`Added support for`
`You can now `
`Improved implementation`
Please change it, you can also refer to any integration release notes for an example.